src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
+import { Todo, TypeError } from '../../types';
+import { getMaxNumberId } from '../../utils/getMaxNumberId';
+import { getUserById } from '../../utils/getUserById';
```

Comment:
[CODE STYLE] - The `getUserById` utility is used to derive data from `usersFromServer`. This should be done in the parent component and passed down as props to maintain separation of concerns and to prepare data in one place.